src/workers/data-verification.ts

```diff
@@ -129,6 +135,13 @@ export class DataVerificationWorker {
           indexedDataRoot,
           computedDataRoot,
         });
+        if (this.dataImporter) {
+          log.debug(
+            'Because of data-root mismatch, we are queueing data item for re-import.',
```

Comment:
Nit: "re-import" should not contain a "-"

Reply:
fixed